getting-started.html

```diff
@@ -319,6 +319,9 @@ <h3 style="color:white">Ready to continue your journey?
                 </div>
                 <div class="col-md-6">
                     <p style="float:right">
+                        <a class="twitter-share-button"
+                            href="https://twitter.com/intent/tweet?text=Creating APIs has never been easier using #LoopBack. Check out LB4 @ v4.loopback.io via @StrongLoop">
```

Comment:
Can we tweak the message a bit to be as follows:

```
Creating APIs has never been easier using #LoopBack 4 by @StrongLoop. Check it out at v4.loopback.io.
```

Same for the one on the bottom.